server.js

```diff
@@ -7395,6 +7395,148 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// PHP version from .travis.yml
+camp.route(/^\/travis-yml\/php-v\/([^/]+\/[^/]+)(?:\/([^/]+))?\.(svg|png|gif|jpg|json)$/,
```

Comment:
This name is confusing, since this is pulling config data from a live Travis API. I think it would be clearer to just call it `/travis/php-v/`.

Reply:
`/travis/php-v/` will conflict with #1255

Reply:
i rename it